staging/src/k8s.io/kms/apis/v2beta1/api.pb.go

```diff
@@ -423,7 +423,7 @@ func (c *keyManagementServiceClient) Status(ctx context.Context, in *StatusReque
 
 func (c *keyManagementServiceClient) Decrypt(ctx context.Context, in *DecryptRequest, opts ...grpc.CallOption) (*DecryptResponse, error) {
 	out := new(DecryptResponse)
-	err := c.cc.Invoke(ctx, "/v2alpha1.KeyManagementService/Decrypt", in, out, opts...)
+	err := c.cc.Invoke(ctx, "/v2beta1.KeyManagementService/Decrypt", in, out, opts...)
```

Comment:
From an API perspective this is the most interesting part of this change - we invoke different versioned methods, similar to how Kube APIs work.  It makes me think that we should drop the `Status.Version` field altogether, but I still do not really understand gRPC APIs / versioning for them.